test/stat_tracker_test.rb

```diff
@@ -84,8 +84,7 @@ def test_average_goals_per_game
      assert_equal 4.22, @stat_tracker.average_goals_per_game
    end
 
-   def test_average_goals_per_season
-     skip
+   def test_average_goals_by_season
```

Comment:
Yeah I saw this one was previously skipped. I'm guessing it didn't work yet? In any case, it works now. Nice job man